src/content/docs/app-development/Intro.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Getting Started
+---
+
+import { Aside,
```

Comment:
Please fix these imports in every file this occurs in

Reply:
(specifically, please do not break them into multiple lines unless absolutely necessary)